x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_overview_logic.test.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { LogicMounter, mockHttpValues, mockFlashMessageHelpers } from '../../../__mocks__';
+
+import { nextTick } from '@kbn/test/jest';
+
+jest.mock('../engine', () => ({
+  EngineLogic: { values: { engineName: 'some-engine' } },
+}));
+
+import { CrawlerOverviewLogic } from './crawler_overview_logic';
+import { CrawlerPolicies, CrawlerRules, CrawlRule } from './types';
+
+const DEFAULT_VALUES = {
+  domains: [],
+};
+
+const DEFAULT_CRAWL_RULE: CrawlRule = {
+  id: '-',
+  policy: CrawlerPolicies.Allow,
+  rule: CrawlerRules.regex,
+  pattern: '.*',
+};
+
+describe('CrawlerOverviewLogic', () => {
+  const { mount } = new LogicMounter(CrawlerOverviewLogic);
+  const { http } = mockHttpValues;
+  const { flashAPIErrors } = mockFlashMessageHelpers;
+
+  beforeEach(() => {
+    mount();
+  });
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
```

Comment:
I suppose it's totally fine if you use `afterEach` here, but 99% of our other tests add `clearAllMocks` in `beforeEach`. You may want to follow suit for consistency unless there is a compelling reason to do this in `afterEach` instead.

Reply:
I've addressed this in 1714daf (and cleaned up two other crawler test files for good measure)